docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -136,17 +148,54 @@ Using the https://mustache.github.io/[Mustache] template syntax `{{variable name
 `alert.actionSubgroup`:: The action subgroup of the alert that scheduled the action.
 `alert.actionGroupName`:: The name of the action group of the alert that scheduled the action.
 `alert.flapping`:: A flag on the alert that indicates whether the alert status is changing repeatedly.
+`kibanaBaseUrl`:: The configured <<server-publicBaseUrl,`server.publicBaseUrl`>>. If not configured, this will be empty.
+
+[float]
+[[alert-summary-action-variables]]
+===== Action variables for summary of alerts
+
+NOTE: This type of action frequency is not available for all rule types.
+
+When the action frequency is **Summary of alerts**, rules pass the following
+variables:
+
+`rule.id`:: The ID of the rule.
+`rule.name`:: The name of the rule.
+`rule.spaceId`:: The ID of the space for the rule.
+`rule.tags`:: The list of tags applied to the rule.
+`date`:: The date the rule scheduled the action, in ISO format.
+`kibanaBaseUrl`:: The configured <<server-publicBaseUrl,`server.publicBaseUrl`>>. If not configured, this will be empty.
+`alerts.all.count`:: The count of all alerts.
+`alerts.all.data`:: An array of objects for all alerts.
+`alerts.new.count`:: The count of new alerts.
+`alerts.new.data`:: An array of objects for new alerts.
+`alerts.ongoing.count`:: The count of ongoing alerts.
+`alerts.ongoing.data`:: An array of objects for ongoing alerts.
+`alerts.recovered.count`:: The count of recovered alerts.
+`alerts.recovered.data`:: An array of objects for recovered alerts.
+
+[float]
+[[alert-data-fields]]
+===== Fields available under alerts.*.data
```

Comment:
> ===== Fields available under alerts.*.data

It isn't clear to me at first glance how this section relates to the other two action variable sections (if at all) and where to find `alerts.*.data`.  Can we provide more context?

Reply:
These are only for the second section `Action variables for summary of alerts` 
it has `alerts.all.data`, `alerts.new.data`, `alerts.ongoing.data` and `alerts.recovered.data`
each one of them (`alerts.*.data`) has the below fields.

is there a way to in dent this block and leave under the second section?

Reply:
IMO the increasing heading levels weren't obvious, so I've nested those properties into each appropriate array object via e6facfb7c296dc543d74a11aa82c6c26e23534e9

Reply:
Looks good, more explicit :)